.github/workflows/pytest-core-nompi.yml

```diff
@@ -38,6 +38,7 @@ jobs:
            pytest-osx-py37-clang-omp,
            pytest-docker-py36-gcc-omp
         ]
+        set: [core, adjoint]
```

Comment:
Can we add it to the job name? Just to make it easier to find in the checks.

Reply:
Yeah, having two jobs with the same name show up in CI isn't great - was looking for some suitable workaround but got disturbed. Will take another look later.

The first thing I did was just to add `pytest-docker-py36-gcc-omp-adjoint` etc to the matrix which is fine for now, but you end up with a long list and it's not very extendible if we want to add further separation in the future.

Ideally we can stick to something similar to this but where the job names are clearly distinguished. As I was writing I had an idea re. this, but will need to test later.

Reply:
Oh, silly me, solution is simple, just modify [this](https://github.com/devitocodes/devito/blob/30c5b6cde649a7d9dd4924ca9a602adb1b83d25f/.github/workflows/pytest-core-nompi.yml#L15)...

Reply:
>Can we add it to the job name? Just to make it easier to find in the checks.

I see that's what you meant now. Multitasking :p